### PR TITLE
added hentaifox presences

### PR DESCRIPTION
**Pull request by Bas1C9**

### CODE REVIEW

File: websites/H/hentaifox/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "Bas1C",
+    "id": "303603391941050370"
+  },
+  "service": "hentaifox",
+  "description": {
+    "tr": "Hentai manga okuma sitesidir.",
+    "en": "Hentai is manga reading site."
```
Comment (Alanexei) [inline websites/H/hentaifox/dist/metadata.json:9]: Try and use a more lengthy description. Look up the website in Google for instance and copy the description text.

File: websites/H/hentaifox/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "url": "hentaifox.com",
+  "version": "1.0.1",
+  "logo": "https://i.imgur.com/OcUEF3h.png",
+  "thumbnail": "https://i.imgur.com/a2zJ6cD.png",
```
Comment (Alanexei) [inline websites/H/hentaifox/dist/metadata.json:14]: Remove the NSFW content from the screenshot. *Use the elements tab in devtools to remove images*

**Comment by Bas1C9** [comment]
Presence proof that working.
http://prntscr.com/tiw2u3
http://prntscr.com/tiw3m8 > http://prntscr.com/tiw387
http://prntscr.com/tiw49e > http://prntscr.com/tiw3vf

File: websites/H/hentaifox/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "name": "Bas1C",
+    "id": "303603391941050370"
+  },
+  "service": "hentaifox",
+  "description": {
+    "tr": "HentaiFox hentai manga ve doujinshi İngilizce çeviri yapan en popüler ücretsiz hentai sitelerinden biridir.",
+    "en": "HentaiFox is one of the most popular free hentai sites around for English translated hentai manga and doujinshi."
+  },
+  "url": "hentaifox.com",
+  "version": "1.0.1",
```
Comment (Bas950) [inline websites/H/hentaifox/dist/metadata.json:12]: version 1.0.0 on release

**Comment by Bas1C9** [comment]
> why do you have 2 hentaifoxes?

I've arranged it.

File: websites/H/hentaifox/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "Bas1C",
+    "id": "303603391941050370"
+  },
+  "service": "hentaifox",
```
Comment (Bas950) [inline websites/H/hentaifox/dist/metadata.json:6]: Change service name and folder name to "HentaiFox"

**Pull request merged**